README.rst

```diff
@@ -158,7 +167,6 @@ get the idea.
 Caveats
 -------
 
-* Race condition if you flag a field as stale in a db transaction that takes longer to complete than the celery job takes to be called (so commit your transactions before invalidating the cache).
```

Comment:
may have to bring back this comment and say that is avoidable by using django 1.9 and setting `CACHED_FIELD_TRANSACTION_AWARE`